hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/supervisor/workload/SubscriptionAssignmentRegistry.java

```diff
@@ -101,6 +94,38 @@ public SubscriptionAssignmentView createSnapshot() {
         return new SubscriptionAssignmentView(snapshot);
     }
 
+    private List<SubscriptionAssignment> readExistingAssignments() {
+        return subscriptionsCache.listActiveSubscriptionNames().stream().flatMap(subscriptionName -> {
```

Comment:
Matter of taste of course, but I made it shorter, hopefully complying to your taste :-)